src/Makefile

```diff
@@ -81,5 +93,13 @@ linux: docker-centos
 	docker run -i -t -v $(GOPATH):/go junegunn/centos-sandbox \
 		/bin/bash -ci 'cd /go/src/github.com/junegunn/fzf/src; make TAGS=static'
 
+ubuntu-android: docker-android
+	docker run -i -t -v $(GOPATH):/go junegunn/$@-sandbox \
+		sh -c 'cd /go/src/github.com/junegunn/fzf/src; /bin/bash'
+
+android: docker-android
+	docker run -i -t -v $(GOPATH):/go junegunn/android-sandbox \
+		/bin/bash -ci 'cd /go/src/github.com/junegunn/fzf/src; GOOS=android make android-build'
+
 .PHONY: all build release test install uninstall clean docker \
 	linux arch ubuntu centos docker-arch docker-ubuntu docker-centos
```

Comment:
@junegunn done (sorry for being MIA, got distracted).

i've been using this for a while, seems to work well.

Unfortunately, the only way to test it is with either an android device, or using the android emulator (this is because go builds for the android target relying on the android specific runtime.. which is not available in, for example, vanilla linux on arm).